code/datums/martial/rising_bass.dm

```diff
@@ -0,0 +1,163 @@
+#define REPULSE_PUNCH_COMBO "GHGH"
+#define FOOT_SMASH_COMBO "HH"
+#define DEFT_SWITCH_COMBO "GDD"
+
+/datum/martial_art/the_rising_bass
+	name = "The Rising Bass"
+	id = MARTIALART_RISINGBASS
+	dodge_chance = 100
+	no_guns = TRUE
+	allow_temp_override = FALSE
+	help_verb = /mob/living/carbon/human/proc/rising_bass_help
+
+/datum/martial_art/the_rising_bass/proc/check_streak(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(findtext(streak,SIDE_KICK_COMBO))
+		streak = ""
+		sideKick(A,D)
+		return 1
+	if(findtext(streak,SHOULDER_FLIP_COMBO))
+		streak = ""
+		shoulderFlip(A,D)
+		return 1
+	if(findtext(streak,REPULSE_PUNCH_COMBO))
+		streak = ""
+		repulsePunch(A,D)
+		return 1
+	if(findtext(streak,FOOT_SMASH_COMBO))
+		streak = ""
+		footSmash(A,D)
+		return 1
+	if(findtext(streak,DEFT_SWITCH_COMBO))
+		streak = ""
+		deftSwitch(A,D)
+		return 1
+	return 0
+
+
+/datum/martial_art/the_rising_bass/proc/sideKick(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(!D.IsKnockdown())
+		var/turf/H
+		switch(A.dir)
+			if(NORTH)
+				H = get_step(get_step(A,NORTH),pick(EAST,WEST))
+			if(EAST)
+				H = get_step(get_step(A,EAST),pick(SOUTH,NORTH))
+			if(SOUTH)
+				H = get_step(get_step(A,SOUTH),pick(EAST,WEST))
+			if(WEST)
+				H = get_step(get_step(A,WEST),pick(SOUTH,NORTH))
+		A.do_attack_animation(D, ATTACK_EFFECT_KICK)
+		D.visible_message("<span class='warning'>[A] kicks [D] in the side, sliding them over!</span>", \
+						  "<span class='userdanger'>[A] kicks you in the side, forcing you to step away!</span>")
+		playsound(get_turf(A), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+		D.apply_damage(5, BRUTE, BODY_ZONE_CHEST)
+		D.Knockdown(60)
+		var/turf/L = H
+		for(var/obj/i in H.contents)
+			if(!istype(i,/mob) && i.density == 1)//(i.anchored == 1 && i.density == 1) || istype(i,/obj/structure) || istype(i,/turf/closed)
+				L = D.loc
+		D.forceMove(L)
+		return 1
+	log_combat(A, D, "side kicked (Rising Bass)")
+	return basic_hit(A,D)
+
+/datum/martial_art/the_rising_bass/proc/shoulderFlip(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(!D.IsKnockdown())
+		var/turf/H
+		switch(A.dir)
+			if(NORTH)
+				H = get_step(A,SOUTH)
+			if(EAST)
+				H = get_step(A,WEST)
+			if(SOUTH)
+				H = get_step(A,NORTH)
+			if(WEST)
+				H = get_step(A,EAST)
+		var/L = H
+		for(var/obj/i in H.contents)
+			if(!istype(i,/mob) && i.density == 1)//(i.anchored == 1 && i.density == 1) || istype(i,/obj/structure) || istype(i,/turf/closed)
+				L = A.loc
+		A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+		D.visible_message("<span class='warning'>[A] flips [D] over their shoulder, slamming them into the ground!</span>", \
+						  "<span class='userdanger'>[A] flips you over their shoulder, slamming you into the ground!</span>")
+		playsound(get_turf(A), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+		D.emote("scream")
+		D.apply_damage(10, BRUTE, BODY_ZONE_CHEST)
+		D.apply_damage(30, BRUTE, BODY_ZONE_HEAD)
+		D.SetSleeping(60)
```

Comment:
So I should use them how?

Reply:
"Set" status effects procs are often used when using adrenals, being ahealed.
Colloquially said, they just throw the old value out of the window. It'd be awkward to have your 20 seconds stun/sleep/knockout etcetera magically set to 2 seconds because of some crappy or acient feature.

Reply:
Ah. So how should I do it?

Reply:
So again, how would I use this? To make them stunned for a set amount of time, or something along those lines.

Reply:
```suggestion
		D.Sleeping(60)
```
Simple as that. Just because other martial arts use SetSleeping(), it's not like it's a good thing to do.

Reply:
```suggestion
		D.Sleeping(60)
```
Simple as that. Just because other martial arts use SetSleeping(), it's not like it's a good thing to do.